### ISSUE TITLE
Status keeps showing "update available"

### ISSUE THREAD

**Issue by radriaanse** [ValveSoftware/SteamVR-for-Linux#206]
**Your system information**
* Steam client version (build number or date): 
* Distribution (e.g. Ubuntu): Arch (5.1.15-zen1-1-zen)
* Graphics driver version (run nvidia-settings): 430.26
* Gist for SteamVR System Information: https://gist.github.com/radriaanse/cddf9b74a2f7d30d63e6eeba82a70d64
* Opted into Steam client beta?: Yes
* Opted into SteamVR beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Launching SteamVR shows the status popup on which there is an overview of the Index components and their status.
The HMD shows as having an update, which appears to be incorrect since clicking it opens the other status panel where there is only one button named "remind me later" and everything else is listed as begin up-to-date.
I did however update everything after unboxing for a first time, but the notification of the HMD kept shown.
Trying to update again proceeds but is done almost instantly and after a restart still keeps showing as "update available".

I did try reattaching the Index, also multiple power cycles of the pc.
Problem is here since launch past Friday.

#### Steps for reproducing this issue:
1. Launce steam (steam-runtime)
2. Start SteamVR
3. See update status of Index in overlay
4. Click update, continue with update
5. After 'update', return to status overlay and notice notification still there.

**Comment by beniwtv** [comment]
Same issue here on Manjaro (kernel 5.2rc4)

**Comment by hellocld** [comment]
Also having this issue on Debian Buster (4.19.0-5-amd64)

**Comment by lostgoat** [comment]
On Linux, the HMD watchman application firmware update works, but the HMD dongle firmware update processes doesn't. (See known issues on the README.md)

We'll fix the notifications so that they only show up when appropriate on Linux. A follow-up update will address Dongle firmware updates for Index on Linux.

**Comment by lostgoat** [comment]
I don't think we have a ticket for this issue, so lets keep this one open to track the progress on the FW update changes.

**Comment by kisak-valve** [comment]
Per "Fixed Index HMD always notifying a firmware update is available." in the [SteamVR 1.6.1](https://steamcommunity.com/games/250820/announcements/detail/1603760509564290978) beta update, the notification half of this issue should have improved.

**Comment by radriaanse** [comment]
@kisak-valve Yes, the notification is indeed gone now; thanks so far!
